.babelrc

```diff
@@ -8,5 +8,7 @@
       "react-hot-loader/babel",
       "transform-object-rest-spread",
       "transform-class-properties",
+      "transform-react-constant-elements",
```

Comment:
The docs say that this should be enabled only in production because it makes warning messages more cryptic. Is there a way to do that cleanly?